core/server/lib/fs/package-json/package-json.js

```diff
@@ -7,6 +7,11 @@ const errors = require('@tryghost/errors');
 const notAPackageRegex = /^\.|_messages|README.md|node_modules|bower_components/i;
 const packageJSONPath = 'package.json';
 
+/**
+ * @typedef {Object} Ii18n
```

Comment:
`Ii18n` -> `I18n`

Reply:
It should be fine, the idea of the extra "i" is to share that it's an interface declaration :ok_hand: 
Thanks for the review :pray: